scripts/globals/spells/bluemagic/asuran_claws.lua

```diff
@@ -25,6 +25,7 @@ function onSpellCast(caster,target,spell)
     local params = {}
     -- This data should match information on http://wiki.ffxiclopedia.org/wiki/Calculating_Blue_Magic_Damage
         params.tpmod = TPMOD_ACC
+        params.attackType = tpz.attackType.PHYSICAL
         params.damageType = tpz.damageType.BLUNT
```

Comment:
Not necessarily a question directed at you, mrhappyasthma (unless you _want_ to take a guess):

Bgwiki has two different classifications for spells whose damage type _I_ would normally consider "blunt" - they have identifiers for both "Blunt" and "Hand-to-hand". In this instance, they identify Asuran Claws as "hand to hand".

Are there theories on why they might do this, and is it something we should be mindful of?

Reply:
@ibm2431 there actually are different dmg resistances between the 2 on mobs. there are only a few mobs where that difference matters however, and I don't know that anyone confirmed it before tossing it onto the wiki.

Reply:
Would we prefer to update this to H2H then?

Reply:
Interestingly this is `Blunt` here: https://ffxiclopedia.fandom.com/wiki/Asuran_Claws

But in the comments they say:

> Hand-to-Hand-based damage.

Reply:
It is marked H2H here: https://www.bg-wiki.com/bg/Asuran_Claws

So I'll go with that since it seems to be the correct one.

Reply:
Done.